helpers/calendar/analytics_helper.py

```diff
@@ -38,6 +38,23 @@ def convert_dates(self, start_date, end_date):
         end_date = (datetime.strptime(end_date, "%b %d %Y") + relativedelta(days=1)).isoformat() + 'Z'  # noqa: E501
         return(start_date, end_date)
 
+    def validate_current_date(self, start_date, end_date):
+        """
+        Checks for today's date
+        """
+        start_date, end_date = CommonAnalytics.convert_dates(
+            self, start_date, end_date)
+
+        now = datetime.now().strftime('%b %d %Y')
+        date_now = (datetime.strptime(now, "%b %d %Y") + relativedelta(days=1)).isoformat() + 'Z'  # noqa: E501
+        start_dt = dateutil.parser.parse(start_date)
```

Comment:
Hello @BonifaseOrwa, kindly throw more light here. Which variable should I look into in particular. I used `start_dt` and `end_dt` because they had been used in the code base, so I thought using the same would make it it easier  for the person reading through the code to follow.

Reply:
start_date is more readable than start_dt

Reply:
Hey @Bonifase , `start_date` had been used.  Maybe I should look for another name. I used `start_dt, because it had been used before in the codebase. ```start_date, end_date = CommonAnalytics.convert_date self, start_date, end_date)```, That line is where I used start_date.

Reply:
Hello @Bonifase , I have worked on this, I have used `user_start_date` as advised. Thanks